app/routes/reductions.js

```diff
@@ -152,12 +201,16 @@ module.exports = function (router) {
       requestBody.red_end_month === '' || requestBody.red_end_month === undefined ||
       requestBody.red_end_day === '' || requestBody.red_end_day === undefined ||
       requestBody.hours === '' || requestBody.hours === undefined) {
-      result = false
-    } else {
-      result = true
+      return false
+    }
+    // Check if the end date is after the start date
+    var reductionStartDate = new Date(requestBody.red_start_year, requestBody.red_start_month - 1, requestBody.red_start_day)
+    var reductionEndDate = new Date(requestBody.red_end_year, requestBody.red_end_month - 1, requestBody.red_end_day)
+    if (reductionEndDate < reductionStartDate) {
+      return false
     }
 
-    return result
+    return true
```

Comment:
Could we keep it like before please? Use single var, one return statement?

Reply:
I changed it as we are doing more validation and didn't want o create a variable if the first validation wasn't successful. However, I can use the variable if that is the desire.